### PR TITLE
Pivotal ID # 182600130: Remove TSV Feature Flag

### PR DESCRIPTION
**Pull request by jhoanmanuelms**
https://www.pivotaltracker.com/story/show/182600130

- Remove TSV feature flag
- Remove code warnings
- Re-structure iTests
- Remove mongo mode property

### CODE REVIEW

File: client/bio-webclient/src/test/kotlin/ac/uk/ebi/biostd/client/api/ExtSubmissionClientTest.kt [hunk 1]
```diff
@@ -159,9 +159,9 @@ class ExtSubmissionClientTest(
     private fun assertHttpEntity(entity: HttpEntity<LinkedMultiValueMap<String, Any>>) {
         val body = entity.body
 
-        assertThat(body).isNotNull()
+        assertThat(body).isNotNull
```
Comment (JuanCamiloRada) [inline client/bio-webclient/src/test/kotlin/ac/uk/ebi/biostd/client/api/ExtSubmissionClientTest.kt:162]: we need to rollback this one, it is a function not a property.

File: submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/service/SubmissionMongoPersistenceQueryService.kt [hunk 2]
```diff
@@ -39,7 +39,7 @@ internal class SubmissionMongoPersistenceQueryService(
     }
 
     override fun findLatestExtByAccNo(accNo: String, includeFileListFiles: Boolean): ExtSubmission? {
-        val findByAccNo = submissionRepo.findLatestByAccNo(accNo)
+        val findByAccNo = submissionRepo.findByAccNo(accNo)
```
Comment (JuanCamiloRada) [inline submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/service/SubmissionMongoPersistenceQueryService.kt:42]: why we have rename this?

**Review by JuanCamiloRada — APPROVED** [review]
LGTM please check minor comments.

File: submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/service/SubmissionMongoPersistenceQueryService.kt (hunk 2 above)
Comment (jhoanmanuelms) [reply submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/service/SubmissionMongoPersistenceQueryService.kt:42]: This wasn't renamed but replaced by a method that does exactly the same

https://github.com/EBIBioStudies/biostudies-backend-services/pull/596/files#diff-0e84528306be7e333417ccb2a28ec6909b7ff9bb50b4787ce5132980b518a3beR19

**Pull request merged**